### PR TITLE
Test for Ada support on SAMV71 runtime

### PR DESCRIPTION
**Pull request by Glamhoth**

### CODE REVIEW

File: runtime_tests/SamV71_Ada/work/pinger/SDL/code/pinger_ri.ads [hunk 1]
```diff
@@ -0,0 +1,23 @@
+--  DO NOT EDIT THIS FILE, IT WILL BE OVERWRITTEN DURING THE BUILD
```
Comment (Lurkerpas) [inline runtime_tests/SamV71_Ada/work/pinger/SDL/code/pinger_ri.ads]: Please remove unnecessary files

File: runtime_tests/SamV71_Ada/work/ponger/implem/default/Ada/src/ponger.adb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+--  !! the procedures corresponding to the provided interfaces in this  !!
+--  !! file. The up-to-date signatures can be found in the .ads file.   !!
+
+
+package body Ponger is
+
+   procedure Ping (pingArg : in out asn1SccMyinteger) is
+   
+      begin
+         --  Write your code here
```
Comment (Lurkerpas) [inline runtime_tests/SamV71_Ada/work/ponger/implem/default/Ada/src/ponger.adb]: Should this be empty?

File: runtime_tests/SamV71_Ada/work/pinger/SDL/code/pinger_ri.ads (hunk 1 above)
Comment (Glamhoth) [reply runtime_tests/SamV71_Ada/work/pinger/SDL/code/pinger_ri.ads]: Removed

File: runtime_tests/SamV71_Ada/work/ponger/implem/default/Ada/src/ponger.adb (hunk 2 above)
Comment (Glamhoth) [reply runtime_tests/SamV71_Ada/work/ponger/implem/default/Ada/src/ponger.adb]: Nope, fixed that

**Review by Lurkerpas — APPROVED** [review]

**Pull request merged**